core/src/main/java/org/apache/iceberg/util/ParallelIterable.java

```diff
@@ -77,7 +78,7 @@ public CloseableIterator<T> iterator() {
     return iter;
   }
 
-  private static class ParallelIterator<T> implements CloseableIterator<T> {
+  static class ParallelIterator<T> implements CloseableIterator<T> {
```

Comment:
we can probably annotate the class as well with `@VisibleForTesting`

Reply:
indeed, that's a miss. thanks